src/domain/matomo/__tests__/__snapshots__/matomo.test.jsx.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PageWrapper matches snapshot 1`] = `
+<div>
+  <div>
+    Children
+  </div>
+</div>
```

Comment:
Should the snapshot look like this? What does this test verify?

Reply:
Yes. Just a very simple test that makes sure that Matomo component renders the children. Not saying it is very important / ciritcal test but doesn't hurt to have it :)